docker/genetics/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM {{ hail_public_image.image }}
+
+RUN apt-get update && \
+  apt-get -y install \
+    cmake \
+    libbz2-dev \
+    libcurl4-openssl-dev \
+    liblzma-dev \
+    zlib1g-dev \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir samtools && \
+    (cd samtools && \
+     curl -LO https://github.com/samtools/samtools/releases/download/1.9/samtools-1.9.tar.bz2 && \
+     tar -xf samtools-1.9.tar.bz2 && \
+     rm -rf samtools-1.9.tar.bz2 && \
+     cd samtools-1.9 && \
+     ./configure --without-curses && \
+     make && \
+     make install)
+RUN git clone --recursive https://github.com/vcflib/vcflib.git && \
+    (cd vcflib && make)
+RUN curl -LO http://s3.amazonaws.com/plink2-assets/plink2_linux_avx2_latest.zip && \
```

Comment:
What's your reasoning for using latest? I would think we'd want to version control what the contents of each image are so someone could cite what version they used.

Reply:
Chris Chang deletes old versions when he posts new versions so we'd have to PR a new URL every couple weeks to fix failing deploys.

Reply:
And I kind of trust plink to not break something 🤷